rskj-core/src/test/java/co/rsk/peg/BridgeSerializationUtilsTest.java

```diff
@@ -1009,6 +1012,27 @@ public void deserializeInteger() {
         Assert.assertEquals(1200, BridgeSerializationUtils.deserializeInteger(RLP.encodeBigInteger(BigInteger.valueOf(1200))).intValue());
     }
 
+    @Test
+    public void serializeScript() {
+        Script expectedScript = ScriptBuilder.createP2SHOutputScript(2, Lists.newArrayList(new BtcECKey(), new BtcECKey(), new BtcECKey()));
+
+        byte[] actualData = BridgeSerializationUtils.serializeScript(expectedScript);
+
+        Assert.assertEquals(expectedScript, new Script(((RLPList) RLP.decode2(actualData).get(0)).get(0).getRLPData()));
+    }
+
+    @Test
+    public void deserializeScript() {
+        Script expectedScript = ScriptBuilder.createP2SHOutputScript(2, Lists.newArrayList(new BtcECKey(), new BtcECKey(), new BtcECKey()));
+        byte[][] rlpElements = new byte[1][];
+        rlpElements[0] = RLP.encodeElement(expectedScript.getProgram());
+        byte[] data = RLP.encodeList(rlpElements);
```

Comment:
Minor comment: we could use  `byte[] data = RLP.encodeList(RLP.encodeElement(expectedScript.getProgram()));` directly instead of declaring explicitly an array of arrays

Reply:
done